### PR TITLE
[ci] set_pll at start of FPGA tests

### PR DESCRIPTION
**Pull request by drewmacrae**
Signed-off-by: Drew Macrae <drewmacrae@google.com>

### CODE REVIEW

**Review by dmcardle — APPROVED** [review]
LGTM % missing flags

**Comment by drewmacrae** [comment]
I don't know whether this used to be performed by the Python scripts in CI or if it was done as part of setting up the other FPGA boards, but doing this on each run should help the BergenBoards to carry a bit less state from previous runs.

**Review by jprendes — APPROVED** [review]
LGTM.
Thanks Drew!
This also simplifies the process of adding new hardware to CI :-)

**Comment by alphan** [comment]
Shouldn't we look into the lifetime of the EEPROM of the PLL and make a rough calculation to understand when we can expect it go bad?

**Comment by drewmacrae** [comment]
> Shouldn't we look into the lifetime of the EEPROM of the PLL and make a rough calculation to understand when we can expect it go bad?

It's rated for 100 cycles so we should be fine, right?

**Comment by alphan** [comment]
> > Shouldn't we look into the lifetime of the EEPROM of the PLL and make a rough calculation to understand when we can expect it go bad?
> 
> It's rated for 100 cycles so we should be fine, right?

🚢

**Comment by alphan** [comment]
@drewmacrae PTAL the last commit.

File: sw/host/opentitanlib/src/transport/cw310/usb.rs [hunk 1]
```diff
@@ -410,6 +410,16 @@ impl Backend {
 
     /// Write a byte to the CDCE906 PLL chip.
     fn pll_write(&self, addr: u8, data: u8) -> Result<()> {
+        // We don't want the EEPROM to wear out prematurely. Write only if `data` is different than
+        // what is already stored.
+        if self.pll_read(addr)? == data {
+            log::debug!(
+                "Skipping PLL write since address {} is already {}"
```
Comment (drewmacrae) [inline sw/host/opentitanlib/src/transport/cw310/usb.rs:417]: we want a comma here

**Review by alphan — APPROVED** [review]

**Comment by drewmacrae** [comment]
FWIW I approve of @alphan's commit

**Pull request merged**